config/database.yml

```diff
@@ -1,25 +1,14 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
   pool: 5
-  timeout: 5000
+  username: <%= `whoami` %>
+  password:
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: haxxor-news_development
 
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
-
-production:
-  <<: *default
-  database: db/production.sqlite3
+  database: haxxor-news_test
```

Comment:
typically this file is added to the `.gitignore` file and not checked in to version control. Reason for that being if there are multiple people working on one project, it's possible that one person configured their database differently and needs their own adapter configuration, also for production projects, checking in sensitive information to git (like credentials to a database) is frowned upon as it means the compromising of your github account leads to the compromising of an app's database.

For your use case, it's fine to check this in, just wanted to let you know what was standard here are Viget.